### ISSUE TITLE
Scaling to other resolutions doesn't work

### ISSUE THREAD

**Issue by tim-hellhake** [zmwangx/rust-ffmpeg#18]
I used the `avi-to-ppm.rs` example and changed the scaler block to
```
let mut scaler = Context::get(
    decoder.format(),
    decoder.width(),
    decoder.height(),
    Pixel::RGB24,
    decoder.width()/2,
    decoder.height()/2,
    Flags::BILINEAR,
)?;
```
If I run the example with [bigbunny](http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4) the resulting images look like this:
![image](https://user-images.githubusercontent.com/17127122/88231220-1e02e400-cc74-11ea-8c66-210113fe4e16.png)
It seems like the original height is used when the new height should be used instead.

**Comment by zmwangx** [comment]
Thanks for the report, you caught a bug in `software::scaling::context::get`. Fixed in 02fc43c23e8ae82517b98b8202cc7748c0b3e86f. And I'll tag a patch release shortly.

**Comment by zmwangx** [comment]
Released v4.3.1.

**Comment by tim-hellhake** [comment]
Thank you very much for the fast reaction!
I hope my description was helpful.
